forge-gui/res/cardsfolder/upcoming/machines_god_effigy.txt

```diff
@@ -0,0 +1,9 @@
+Name:Machine God's Effigy
+ManaCost:4
+Types:Artifact
+K:ETBReplacement:Copy:DBCopy:Optional
+SVar:DBCopy:DB$ Clone | Choices$ Creature.Other | AddTypes$ Artifact | AddAbilities$ ABMana | RemoveCardTypes$ True | SpellDescription$ You may have CARDNAME enter the battlefield as a copy of any creature on the battlefield, except it's an artifact and it has "{T}: Add {U}." (It's not a creature.)
```

Comment:
Not sure `.Other` should be here?

Reply:
I'm curious – when you opt to use this replacement effect, how many mana abilities does the result have?

Reply:
copied Birds of paradise 
![image](https://user-images.githubusercontent.com/67333662/198885282-f35a719f-a0f6-4023-932e-4f23289de530.png)